README.md

```diff
@@ -1,26 +1,40 @@
 
 # Tutorials
 
+
 ## An introduction to the use of TERRA REF data and software
 
-Many of these tutorials have complex dependencies. In addition, many datasets are very large. These tutorials use the National Data Service Labs Workbench (NDS Labs Workbench). The NDS Labs Workbench provides the following:
+This repository provides a set of tutorials that are divided by data types and use cases. 
+
+In the repository, you will find three folders that contain examples of how to access data:
+
+* traits
+* sensors
+* plantCV
+
+Within each folder there are both R markdown and Jupyter notebooks. These describe different approaches to accessing data. These are intended to cover diverse use cases, and you will find information about accessing data from web interfaces but the primary focus is on accessing data using R, Python, SQL, and REST APIs. These are intended to provide quick-start introductions to access data along with computing environments required for further exploration. They are not intended to teach analyses, although some illustrative visualizations and statistical models are provided.
+
+This is a work in progress, and an open source community that welcomes contributions in many forms. Please feel welcome to ask questions, provide suggestions or share analyses that may be of interest to others.
+
+
+## Getting Started
+
+### Requirements
 
-* Custom configured computing environments  
-  * In principle, it is possible to use any docker container on [Docker Hub](https://hub.docker.com/explore/).
-* To get started, see the [workbench/ndslabs_workbench_intro.html](https://htmlpreview.github.io/?https://github.com/terraref/tutorials/blob/master/workbench/ndslabs_workbench_intro.html) tutorial.
-* This will set you up to begin tutorials found in the traits, sensors, and plantCV folders.
+All of the tutorials have been designed to work in the cloud and can be accessed using a web browser. Therefore, the _only technical requirements_ are:
+* Web browser
+* Internet connection
 
```

Comment:
This is likely where I would add a "Prerequisites" section, if I were to add one.